src/Entity/Programme.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\Collection;
+
+class Programme
+{
+    private int $id;
+
+    public string $name = '';
+
+    public string $description = '';
+
+    private \DateTime $startTime;
+
+    private \DateTime $endTime;
+
+    private ?User $trainer;                                 //can be null
+
+    private Room $room;
+
+    private ?Collection $customers;
```

Comment:
The collection shouldn't be nullable, I don't exactly know why but we will be taught these days :)

Reply:
Collection is an array of customers... no customers ... empty array not null